docs/ref/models/database-functions.txt

```diff
@@ -23,8 +23,13 @@ We don't usually recommend allowing ``null=True`` for ``CharField`` since this
 allows the field to have two "empty values", but it's important for the
 ``Coalesce`` example below.
 
+.. _comp_conv-functions:
+
+Comparison and Conversion Functions
```

Comment:
For the casing, I think "Comparison and conversion functions" is more consistent with other documentation (e.g. "expressions" is mostly lower cased in expressions.txt headings).